nautobot/extras/models/jobs.py

```diff
@@ -762,6 +762,13 @@ class ScheduledJob(BaseModel):
         verbose_name="Approval date/time",
         help_text="Datetime that the schedule was approved",
     )
+    custom = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
```

Comment:
We shouldn't ever use `null=True` on `CharField`. (I know there are existing examples of this, but I'd like to not see it propagate. Thanks!)

```suggestion
        blank=True,
```